pkg/proxyagent/agent/agent.go

```diff
@@ -263,10 +266,13 @@ func GetClusterProxyValueFunc(
 
 		servicesToExpose := removeDupAndSortServices(managedProxyServiceResolverToFilterServiceToExpose(serviceResolverList.Items, managedClusterSetMap, cluster.Name))
 
-		// add default agentIdentifiers
 		var aids []string
-		aids = append(aids, fmt.Sprintf("host=%s", cluster.Name))
-		aids = append(aids, fmt.Sprintf("host=%s.%s", cluster.Name, config.AddonInstallNamespace))
+
+		// add default kube-apiserver agentIdentifiers
+		if enableKubeApiProxy {
```

Comment:
@qiujian16 Here, we only add kube-apiserver as `agentIndentifiers` when enableKubeApiProxy equals `true`.